.github/workflows/test.yml

```diff
@@ -150,6 +132,24 @@ jobs:
             python_arch: 'x64'
             tox_env: 'py310-pytest71-xdist250-coverage64'
             os: 'macos-latest'
+          - name: 'py311-pytest72-xdist320-coverage65 (ubuntu)'
+            python: '3.11'
+            toxpython: 'python3.11'
+            python_arch: 'x64'
+            tox_env: 'py311-pytest72-xdist320-coverage65'
+            os: 'ubuntu-latest'
+          - name: 'py311-pytest72-xdist320-coverage65 (windows)'
+            python: '3.11'
+            toxpython: 'python3.11'
+            python_arch: 'x64'
+            tox_env: 'py311-pytest72-xdist320-coverage65'
+            os: 'windows-latest'
+          - name: 'py311-pytest72-xdist320-coverage65 (macos)'
+            python: '3.11'
+            toxpython: 'python3.11'
+            python_arch: 'x64'
+            tox_env: 'py311-pytest72-xdist320-coverage65'
+            os: 'macos-latest'
           - name: 'pypy37-pytest71-xdist250-coverage64 (ubuntu)'
```

Comment:
Re: PyPy dropping PyPy3.7 (https://www.pypy.org/posts/2022/12/pypy-v7311-release.html), should we remove these three pypy-3.7 jobs?

Reply:
I am willing to go either way.

Reply:
Removed.